app/postcss.config.js

```diff
@@ -1,6 +1,23 @@
+import autoprefixer from "autoprefixer";
+import tailwindcss from "tailwindcss";
+import doiuse from "doiuse";
+
 export default {
-    plugins: {
-        tailwindcss: {},
-        autoprefixer: {},
-    },
+    plugins: [
+        tailwindcss(),
+        autoprefixer(),
+        doiuse({
+            browsers: [
+                "chrome >= 109",
+                "firefox >= 109",
+                "edge >= 109",
+                "opera >= 95",
+                "safari >= 16.3",
+                "> 1%",
+            ],
+            onFeatureUsage: (usageInfo) => {
+                throw new Error(`Unsupported CSS Detected: ${usageInfo.featureData.title}`);
+            },
```

Comment:
Personally I'd argue that using unsupported CSS features is not that problematic, as long as you're mindful of them and think of fallbacks for browsers that don't support that feature. One popular mobile browser is Opera Mini for example, which deliberately doesn't support many features to save bandwidth or processing power. In that case I'd say it's expected the site might have a degraded experience, even though it should still be usable.